docroot/modules/custom/dhsc_result_viewer/templates/dhsc-results-list-assured-solutions.html.twig

```diff
@@ -1,5 +1,17 @@
 {% set spriteSrc = base_path ~ directory ~ '/stories/01-atoms/svg/sprite.svg' %}
 
+{% set icon %}
```

Comment:
@matt-lambert-TPX looks like icon is set twice in there!

Reply:
@cedrichillion-manifesto, yeah, the same variable is used for both icons and works because of the order. One is for the back link chevron. The other is for the triangle used on the tool results. I tested this as working but can rename the variable. Cheers!